docs/featurizers.rst

```diff
@@ -223,12 +229,6 @@ BPSymmetryFunctionInput
 .. autoclass:: deepchem.feat.BPSymmetryFunctionInput
   :members:
 
-OneHotFeaturizer
```

Comment:
We should add the `OneHotFeaturizer` docs back in now that we have the generalized version